src/frontend/src/handler/create_mv.rs

```diff
@@ -45,6 +47,22 @@ pub fn gen_create_mv_plan(
         binder.bind_query(*query)?
     };
 
+    if let BoundSetExpr::Select(select) = &bound.body {
+        if select
+            .select_items
+            .iter()
+            .zip_eq(select.aliases.iter())
+            .any(|(select_item, alias)| {
+                matches!(select_item, ExprImpl::AggCall(_)) && alias.is_none()
+            })
+        {
+            return Err(ErrorCode::BindError(
+                "An alias must be specified for an aggregation function".to_string(),
```

Comment:
IMO, it should also apply to normal function.

Reply:
But not necessary in this PR

Reply:
And also literals... Actually the only thing to allow without explicit alias is `InputRef`, which is already getting implicit alias assigned in bind_project. Maybe we can simply check that all `aliases` in `BoundSelect` is not `None` here.

Reply:
Got it & updated. I misunderstood the `alias` in `BoundSelect`, and thought it's only user's inputs.